special-calculations/economic.js

```diff
@@ -30,23 +30,32 @@ module.exports = [
   // earnings
   {
     variable: 'mdewrk',
-    type: 'median',
+    specialType: 'median',
```

Comment:
Outta curiosity, why the schema change?

Reply:
no schema change, schema alignment. All of them should be `specialType`